js/friday.js

```diff
@@ -1,34 +1,38 @@
 (function IsItFridayYet() {
   const date = new Date();
-  const theMonth = date.getMonth();
-  const theDay = date.getDay();
-  const element = document.getElementById("friday");
-  const spans = document.getElementsByClassName("fallingElement");
+  const thisMonth = date.getMonth();
+  const thisDay = date.getDay();
+  const theBody = document.body;
+  const textElement = document.getElementById("friday");
   let season = "";
 
-  if (theDay === 5) {
-    element.innerHTML = "YES";
+  // return a programatic yes on Fridays to print on homepage.
+  function isFriday(n) {
+    return (n == 5 ? "YES" : "NO");
+  }
 
-    switch (true) {
-      case theMonth >= 3 && theMonth < 6:
-        season = "spring";
-        break;
-      case theMonth >= 6 && theMonth < 9:
-        season = "summer";
-        break;
-      case theMonth >= 9 && theMonth < 12:
-        season = "autumn";
-        document.body.style.backgroundColor = "fff7c8";
-        break;
-      default:
-        season = "winter";
-        document.body.style.backgroundColor = "dcf0f4";
-        break;
+  // return a season string to switch up body class later.
+  function thisSeason(range) {
+    if(range >= 2 && range < 5 ) {
+      return "spring";
+    } else if(range >= 5 && range < 8 ) {
+      return "summer";
+    } else if(range >= 8 && range < 11) {
+      return "fall";
+    } else if(range >= 11 || range <= 2) {
+      return "winter";
+    } else {
+      return "";
     }
-    Array.from(spans).forEach(function (element) {
-      element.classList.add(season);
-    });
-  } else {
-    element.innerHTML = "NO";
   }
+
```

Comment:
My code comments can use some additional fixes but the goal of our JS comments should always answer the question of WHY we are doing something, never the what. The what is what the code answers. The reasoning for why is true comment mastery. It will give our collaborators context and that context can allow them to scrub through the code efficiently and in some cases give them the context to provide an even stronger solution than the one you created.